python/python/lance/dataset.py

```diff
@@ -152,7 +152,7 @@ def when_not_matched_by_source_delete(self, expr: Optional[str] = None):
 
 
 class LanceDataset(pa.dataset.Dataset):
-    """A dataset in Lance format where the data is stored at the given uri."""
+    """A Lance Dataset in Lance format where the data is stored at the given uri."""
```

Comment:
This changes makes things a bit redundant.  Maybe just `A Lance Dataset where the data is stored at the given uri.`